src/test/java/uk/gov/pay/directdebit/payments/services/PaymentViewServiceTest.java

```diff
@@ -51,17 +60,39 @@ public void setUp() {
                     PaymentState.PROCESSING_DIRECT_DEBIT_DETAILS);
             paymentViewList.add(paymentView);
         }
-        paymentViewService = new PaymentViewService(paymentViewDao);
-        when(paymentViewDao.searchPaymentView(gatewayAccountExternalId, 0l, 100l)).thenReturn(paymentViewList);
+        paymentViewService = new PaymentViewService(paymentViewDao, gatewayAccountDao);
+        gatewayAccount = new GatewayAccount(1l, gatewayAccountExternalId, null, null, null, null, null);
```

Comment:
We normally use fixtures for this, but it's all right, we can refactor in another pr